index.js

```diff
@@ -21,33 +18,25 @@ fs.mkdir(pathName, { recursive: true }, (error) => {
 
 fs.readFile(file, 'utf8', (err, data) => {
     if (err) {
-        return console.log(err)
```

Comment:
same as above about the error, but more specific to this line: unless you're deliberately trying to be terse, don't return like this. logging the error and short circuiting out of the function are two separate thoughts/instructions, put them on separate lines.

In this case I think its better to throw anyway but if you really wanted to do these two things put them on separate lines.